applications/nrf_desktop/src/modules/dfu_mcumgr.c

```diff
@@ -57,42 +58,37 @@ static void dfu_timeout_handler(struct k_work *work)
 	}
 }
 
-static enum mgmt_cb_return mcumgr_img_mgmt_cb(uint32_t event,
-					      enum mgmt_cb_return prev_status,
-					      int32_t *rc, uint16_t *group,
-					      bool *abort_more, void *data,
-					      size_t data_size)
+static enum mgmt_cb_return smp_cmd_recv(uint32_t event, enum mgmt_cb_return prev_status,
+					int32_t *rc, uint16_t *group, bool *abort_more,
+					void *data, size_t data_size)
 {
-	LOG_DBG("MCUmgr Image Management Event with the %d ID",
-		u32_count_trailing_zeros(MGMT_EVT_GET_ID(event)));
+	LOG_DBG("MCUmgr SMP Command Recv Event");
```

Comment:
The log could go after the `if (event != MGMT_EVT_OP_CMD_RECV)` (then you know for sure what event is this)